sql/src/test/java/io/crate/operation/reference/sys/node/local/SysNodesExpressionsOnHandlerTest.java

```diff
@@ -303,17 +302,13 @@ public void testProcess() throws Exception {
         collectExpression = resolver.getImplementation(refInfo);
         collectExpression.setNextRow(context);
 
-        Map<String, Object> expectedCpu = new HashMap<>(4);
-        expectedCpu.put("percent", (short) 50);
-        expectedCpu.put("system", 1000L);
-        expectedCpu.put("user", 500L);
-
         Map<String, Object> v = (Map<String, Object>) collectExpression.value();
-        if (isRunningOnWindows() == false) {
-            assertThat((long) v.get("open_file_descriptors"), greaterThan(2L));
-            assertThat((long) v.get("max_open_file_descriptors"), greaterThan(2L));
-        }
-        assertThat(v.get("cpu"), Matchers.is(expectedCpu));
+        assertThat(v.get("open_file_descriptors"), is(1L));
```

Comment:
why has this changed (and the next line)?

Reply:
the SysNodesExpressionsOnHandlerTest uses dummy stats:

previsously ProcessStats where iobtained from the ProcessService with empty settings:
https://github.com/crate/crate/pull/6625/files/16919385913fed9afb86f2f45b1dfbfbf1face3e#diff-064084266708e2f9f7f5f0faebadbde2L93

now we used the dummy values:
https://github.com/crate/crate/pull/6625/files/16919385913fed9afb86f2f45b1dfbfbf1face3e#diff-064084266708e2f9f7f5f0faebadbde2R94
this is easier for testing IMO

Reply:
ok. Thx!